test/Buddy/functional/ShowQueriesTest.php

```diff
@@ -0,0 +1,99 @@
+<?php declare(strict_types=1);
+
+/*
+ Copyright (c) 2022, Manticore Software LTD (https://manticoresearch.com)
+
+ This program is free software; you can redistribute it and/or modify
+ it under the terms of the GNU General Public License version 2 or any later
+ version. You should have received a copy of the GPL license along with this
+ program; if you did not, you can find it at http://www.gnu.org/
+ */
+
+use Manticoresearch\BuddyTest\Trait\TestFunctionalTrait;
+use PHPUnit\Framework\TestCase;
+
+class ShowQueriesTest extends TestCase {
+
+	use TestFunctionalTrait;
+
+	/**
+	 * @var string $testTable
+	 */
+	protected string $testTable;
+
+	public function testSQLShowQueriesOk(): void {
+		echo "\nTesting the execution of SQL SHOW QUERIES statement\n";
+		if (!self::hasMySQL()) {
+			echo "MySQL is not installed\n";
+			$this->markTestSkipped();
+		}
+		$query = 'SHOW QUERIES';
+		$out = static::runSqlQuery($query);
+		$result = 'id	query	protocol	host';
+		$this->assertEquals($result, $out[0]);
+		$query = ' show queries ';
+		$out = static::runSqlQuery($query);
+		$this->assertEquals($result, $out[0]);
+	}
+
+	public function testSQLShowQueriesFail(): void {
+		echo "\nTesting the fail on the execution of SQL SHOW QUERIES statement\n";
+		if (!self::hasMySQL()) {
+			echo "MySQL is not installed\n";
+			$this->markTestSkipped();
+		}
+		$query = 'SHOW QUERIES 123';
+		$out = static::runSqlQuery($query);
+		$result = [
+			'ERROR 1064 (42000) at line 1: sphinxql: syntax error, unexpected identifier, '
+			. "expecting VARIABLES near 'QUERIES 123'",
+		];
+		$this->assertEquals($result, $out);
+	}
+
+	public function testHTTPShowQueriesOk(): void {
+		echo "\nTesting the execution of HTTP SHOW QUEIRES statement\n";
+		if (!self::hasCurl()) {
```

Comment:
I think we should remove this check and make sure that all required are installed on os